examples/test-app/src/integration-test/groovy/repoapp/OrgSpec.groovy

```diff
@@ -452,4 +453,10 @@ class OrgSpec extends Specification {
         list.size() == 11
 
     }
+
+    def "test quick search fields"() {
+        expect:
+        Student.quickSearchFields == ["name", "jumper.name"]
```

Comment:
@alexeyzvegintcev comment on how 'jumper.name' end up here. How are we testing STudent in OrgSpec? its confusing.